### PR TITLE
soleveset `title_doc` changed

### LINKED ISSUE #9610: Change title doc string of solveset Module
The title of **`sympy.solvers.solveset`** module says:

```
This module contains functions to solve a single equation for a single variable.
```
Since, now `linsolve` has been added to `solveset`, so it supports linear `Multivariate`  as well, so the title should be changed appropriately.

### PR DESCRIPTION
**Pull request by gxyd**
Fixes #9610

### CODE REVIEW

**Comment by leosartaj** [comment]
Copy of #9648. We should ask @rjlasota if he will complete #9648 otherwise it's ok.

**Comment by gxyd** [comment]
Sorry. I guess i should have pinged @rjlasota before i submit mine.

**Comment by rjlasota** [comment]
You go ahead. I'm busy with other projects now.

File: sympy/solvers/solveset.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 """
-This module contains functions to solve a single equation for a single variable.
+This module contains functions to:
+
+    - solve a single equation for a single variable, in any domain either real or complex.
+    - solve a system of linear equations with N variables and M equations.
```
Comment (aktech) [inline sympy/solvers/solveset.py]: A newline between these would be good.

**Comment by aktech** [comment]
+1

**Comment by aktech** [comment]
Thanks, Merging.

**Comment by gxyd** [comment]
Thanks @aktech

**Pull request merged**